app/assets/stylesheets/navbar.scss

```diff
@@ -19,10 +19,14 @@ $side-red: #e56c69;
   bottom: 0;
   width: 100%;
   height: auto;
-  margin-top: 50px;
+  padding: 10px;
   background-color: $site-white;
 }
 
+.footer a{
```

Comment:
Merge rule `.footer a` with rule on line 17<br>Opening curly brace `{` should be preceded by one space